packages/components/Callout/src/Callout.tsx

```diff
@@ -34,7 +33,7 @@ export const Callout = compose<ICalloutType>({
   },
   settings: settings,
   slots: {
-    root: RCTCallout
+    root: NativeComponent('RCTCallout')
```

Comment:
It's probably worth checking that the changes you're making here by wrapping the native components with a functional component don't interfere with the setAndForwardRef stuff @PPatBoyd had to do to get useViewCommandFocus working.

Reply:
In truth they get updated to directly use the host via the `lateBindComponent` call that happens internally. The component wrappers are only point in time wrappers.